tests/reporter/kubernetes.go

```diff
@@ -659,18 +656,15 @@ func (r *KubernetesReporter) logPVCs(virtCli kubecli.KubevirtClient) {
 	fmt.Fprintln(f, string(j))
 }
 
-func (r *KubernetesReporter) logLogs(virtCli kubecli.KubevirtClient, since time.Time) {
+func (r *KubernetesReporter) logLogs(virtCli kubecli.KubevirtClient, logsdir string, pods *v1.PodList, since time.Time) {
```

Comment:
maybe change `logsDir` to `podsDir`? not sure...

Reply:
It was logsdir before, and allover, which is pretty fine to use the same name in all functions i think, as its indeed logsdir, no need to be specific imo